### ISSUE TITLE
Requesting permission to fix a few issues and perhaps simplify the code.

### ISSUE THREAD

**Issue by EdThePro101** [renatoathaydes/ansi-color#3]
Hey there!

I was maybe wondering whether I could fix a few issues and/or tidy up the documentation/source code. I see in the `display.rkt` module that there were plans to implement `font-style`, but it was never `provide`d for use. Also the `with-colors` statement doesn't seem to support the `font-style` parameter.

There is also a definition called `ansi-color?`, but I don't seem to see any way a user could specify an integer for the color.

Would it be possible if I could attempt to make a few changes and additions and open a PR?

**Comment by renatoathaydes** [comment]
Yes I stopped developing this a long time ago. Feel free to make a PR.
